src/custom/components/Tokens/TokensTableRow.tsx

```diff
@@ -189,19 +185,12 @@ const DataRow = ({
       <Cell>{fiatValue}</Cell>
 
       <Cell>
-        <Link to={tradeLink(tokenData, OrderKind.BUY)}>
-          <TableButton color={theme.green1}>Buy</TableButton>
-        </Link>
-      </Cell>
-
-      <Cell>
-        <Link to={tradeLink(tokenData, OrderKind.SELL)}>
-          <TableButton color={theme.red1}>Sell</TableButton>
-        </Link>
+        <ExtLink href={getBlockExplorerUrl(chainId, tokenData.address, 'token')}>
+          <TableButton>Contract</TableButton>
```

Comment:
I find this a bit strange

I would actually expect the opposite, click on the token would take me to Etherscan, and the link would read as "Trade".

not super strong opinion though